src/proto/mgmt/srv.proto

```diff
@@ -70,6 +73,11 @@ message GetAttachInfoResp {
 		string uri = 2;
 	}
 	repeated Psr psrs = 2;	// CaRT PSRs of the system group.
+	string Provider = 3;
+	string Interface = 4;
+	string Domain = 5;
+	uint32 CrtCtxShareAddr = 6;
+	uint32 CrtTimeout = 7;
```

Comment:
Would be good to add comments for these new fields

Reply:
Done.